docs-samples/data-science/semantic-link-samples/great_expectations_tutorial.ipynb

```diff
@@ -0,0 +1,2642 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "2c8b374e-9883-4f6f-9df8-d068ee21d8e5",
+      "metadata": {},
+      "source": [
+        "# Tutorial: Use Great Expectations to validate  Power BI datasets\n",
+        "This tutorial illustrates how to use Semantic Link together with [Great Expectations](https://greatexpectations.io/) to perform data validation."
```

Comment:
Are we updating the docs to use the lowercase version of semantic link?

Reply:
yes we should.